proposals/low-level-struct-improvements.md

```diff
@@ -136,21 +136,20 @@ The rules we define for `ref` fields must ensure the `Span<T>` constructor prope
 
 <a name="ref-fields-escape"></a>
 
-To accomplish this the span safety rules will be changed as follows. First for the constructor of a `ref struct` that directly contains a `ref field`: 
-- If the constructor is annotated with `[RefFieldEscapes]` then `ref` fields can be initialized with any value that has *ref-safe-to-escape* to the *calling method*
-- Else `ref` fields can only be initialized with known heap values.
+To accomplish this the span safety rules for method invocation will be changed. The method invocation rules presently state:
 
-These rules ensure that the caller can understand when constructors will or will not capture parameters by `ref`. 
+> An rvalue resulting from a method invocation e1.M(e2, ...) is *safe-to-escape* from the smallest of the following scopes:
+> 1. The entire enclosing method
+> 2. The *safe-to-escape* of all argument expressions (including the receiver)
+
+The change will be in how the *safe-to-escape* scope is calculated for arguments of method invocation when the return type is a ref-like type. The updated rule will read as follows:
```

Comment:
Why does the *safe-to-escape* scope change for the arguments to the method invocation, rather than the *safe-to-escape* scope of the arguments affecting the *safe-to-escape* scope of the ref-like return value?